model/pdata/generated_metrics_test.go

```diff
@@ -2007,7 +2014,9 @@ func fillTestNumberDataPoint(tv NumberDataPoint) {
 	fillTestStringMap(tv.LabelsMap())
 	tv.SetStartTimestamp(Timestamp(1234567890))
 	tv.SetTimestamp(Timestamp(1234567890))
-	tv.SetValue(float64(17.13))
+	tv.SetDoubleVal(float64(17.13))
+	tv.SetIntVal(int64(17))
```

Comment:
Should this be only one set?

Reply:
SetDoubleVal for now, added a TODO to remember to test with both int and double values in the future.